types/errors/errors.go

```diff
@@ -106,6 +106,9 @@ var (
 	// ErrInvalidType defines an error an invalid type.
 	ErrInvalidType = Register(RootCodespace, 29, "invalid type")
 
+	// ErrTxTimeoutHeight defines an error for when a tx timesout due to it's height
```

Comment:
timesout -> times out

Reply:
ACK.

nit: just make a code suggestion next time 👍

Reply:
It's hard on mobile sorry...